tool/tsh/help.go

```diff
@@ -32,7 +32,10 @@ EXAMPLES:
   $ tsh --proxy=host.example.com login two
 
   Select cluster "two" using existing credentials and proxy:
-  $ tsh login two`
+  $ tsh login two
+
+  For passwordless access use:
+  $ tsh --auth=passwordless login`
```

Comment:
It probably works like this, but docs have the other way around
```
tsh login ... --auth=passwordless
```
https://goteleport.com/docs/access-controls/guides/passwordless/#step-22-authenticate

I think it could be useful to state that this only works for `local` auth

Reply:
Also note that `--auth=passwordless` is not required if the cluster is set to default to passwordless.

Reply:
@marcoandredinis @zmb3 PTAL, I have added in `Notes` section info about default and local auth.